code/modules/mob/living/living.dm

```diff
@@ -818,7 +818,7 @@
 			who.unEquip(what)
 			if(silent)
 				put_in_hands(what)
-			add_attack_logs(src, who, "Stripped of [what]", isLivingSSD(who))
+			add_attack_logs(src, who, "Stripped of [what]", isLivingSSD(who) ? null : ATKLOG_ALMOSTALL)
```

Comment:
These should both probably be `MOST` since stealing stuff or planting stuff on people is a somewhat big deal.